OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -390,6 +400,8 @@ protected override MovePart OnComplete(Actor self, Mobile mobile, Move parent)
 							Util.GetNearestFacing(mobile.Facing, self.World.Map.FacingBetween(mobile.ToCell, nextCell.Value.First, mobile.Facing)),
 							moveFraction - MoveFractionTotal);
 
+						// These two lines are doing exactly the same as mobile.SetPosition, except for setting the Visual Position.
```

Comment:
What is the problem here?  The position and cell location are conceptually different things, and get updated at different times.